packages/core/src/assembler/finalizeRelationships.spec.ts

```diff
@@ -94,7 +97,10 @@ describe("finalizeRelationships", () => {
           ...schemas.Todo,
           attributes: {
             ...schemas.Todo.attributes,
-            userId: uuid().finalize(),
+            userId: {
+              ...uuid({ hidden: true }).finalize(),
+              name: "uuid()",
```

Comment:
this is related to your previous comment of not relying on that truthy `targetAttributeValue`

the reason it's overriding name here (and a couple other spots) is because i am using `uuid({ hidden: true }).finalize()` for testing comparison; internally it is not calling `uuid({ hidden: true })`, it's calling `attribute ?? uuid()` and therefore the name is different; `uuid({ hidden: true })` vs `uuid()`